service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/NotificationServiceTest.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.hmcts.reform.fpl.service.email;
+
+import com.google.common.collect.ImmutableMap;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.service.notify.NotificationClient;
+import uk.gov.service.notify.NotificationClientApi;
+import uk.gov.service.notify.NotificationClientException;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static uk.gov.hmcts.reform.fpl.NotifyTemplates.PARTY_ADDED_TO_CASE_BY_EMAIL_NOTIFICATION_TEMPLATE;
+
+@ExtendWith(SpringExtension.class)
+public class NotificationServiceTest {
+    public static final String TEST_RECIPIENT_EMAIL = "test@example.com";
+    public static final String REFERENCE = "12345L";
+    public static final String TEMPLATE_ID = PARTY_ADDED_TO_CASE_BY_EMAIL_NOTIFICATION_TEMPLATE;
+
+    @Mock
```

Comment:
why we need this, NotificationService does not depend on this one. If this is dependedncy of NotificationClient then it does not matter as NotificationClient is mocked here. Please remove